dokka-subprojects/plugin-base/src/main/kotlin/org/jetbrains/dokka/base/transformers/pages/annotations/SinceKotlinTransformer.kt

```diff
@@ -37,20 +37,29 @@ public class SinceKotlinVersion(str: String) : Comparable<SinceKotlinVersion> {
     }
 
     override fun toString(): String = parts.joinToString(".")
+
+    internal companion object {
+        internal const val SINCE_KOTLIN_TAG_NAME = "Since Kotlin"
+
+        private val minSinceKotlinVersionOfPlatform = mapOf(
+            Platform.common to SinceKotlinVersion("1.0"),
+            Platform.jvm to SinceKotlinVersion("1.0"),
+            Platform.js to SinceKotlinVersion("1.1"),
+            Platform.native to SinceKotlinVersion("1.3"),
+            Platform.wasm to SinceKotlinVersion("1.8"),
+        )
+
+        fun minSinceKotlinVersionOfPlatform(platform: Platform): SinceKotlinVersion {
```

Comment:
nit: since this was moved into `SinceKotlinVersion`, I think it can be renamed to be just `minVersionOfPlatform`, so that the call looks like `SinceKotlinVersion.minVersionOfPlatform()` instead of duplicating `SinceKotlinVersion.minSinceKotlinVersionOfPlatform()`